lib/sms.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+
+class Sms
+  def order_confirmation
+    # account_sid = "AC9cbe834cb7d3797afe4bd7518ef3d1f8"
+    # auth_token = "ff765c0b821d5b80e2b73b4bcfdbe1f6"
+    #
+    # @client = Twilio::REST::Client.new(account_sid, auth_token)
+    #
+    # @client.api.account.messages.create(
+    #   to: "+447549420236",
+    #   from: ENV["TWILIO_NUMBER"],
+    #   body: "Your order has been received. it will arrive at #{Time.now + 60}"
```

Comment:
Not sure your Time.now + 60 will work. The base units for time in ruby is seconds. So you'll need to add 60 * 60 for it be an hour later.